File: projeto/src/app/model/aluno.ts
export class Aluno {
    idAluno: number;
    nome: String;
    cpf: string;
    email: string;
    senha: string;
    telefone: string;
    uf: string;
    cep: string;
    cidade: string;
    bairro: string;
    rua: string;
    numero: number;
    foto: string;

    constructor(){
        this.idAluno = 0;
        this.nome = "";
        this.cpf = "";
        this.email = "";
        this.senha = "";
        this.telefone = "";
        this.uf = "";
        this.cep = "";
        this.cidade = "";
        this.bairro = "";
        this.rua = "";
        this.numero = undefined;
        this.foto = "";
    }
}

File: api/src/main/java/br/cefet/tutorParticular/model/Aluno.java
package br.cefet.tutorParticular.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Aluno {
    private int idAluno;
    private String nome;
    private String cpf;
    private String email;
    private String senha;
    private String telefone;
    private String uf;
    private String cep;
    private String cidade;
    private String bairro;
    private String rua;
    private int numero;
    private String foto;
}